constants/constants.go

```diff
@@ -5,6 +5,8 @@ import "time"
 const (
 	HealthCheckInterval = 3 * time.Second
 	APITimeoutDuration  = 10 * time.Second
-	DefaultPort         = 9650
+	DefaultAPIPort      = 9650
+	DefaultStakingPort  = 9651
```

Comment:
Yeah every node gets its own port, so as long as we as users don't provide fancy custom ports, it can be fairly reasonably assumed that the operator will set the default

Reply:
I don't think we should hardcode the ports like this. The user could give a different API port in a node's config file, and then we won't be able to make API calls to a given node since we assume they're listening on 9650, right?

Reply:
Really the moment we use kubernetes the user should not think about ports at all. Every node gets its own pod and there is simply no reason to change the port...

Having said that, the other question is if the operator is actually passing any custom ports to the node and actually assigning such a port. It might just be assigning the defaults and and not allowing any others for networking reasons. We'll have to ask @yevhenvolchenko about this.

Reply:
operator will ignore both `DefaultAPIPort` and `DefaultStakingPort`
ref: https://github.com/ava-labs/avalanchego-operator/blob/main/controllers/avalanchego_controller.go#L90

actual values indeed are 9650 and 9651
ref: https://github.com/ava-labs/avalanchego-operator/blob/main/controllers/avalanchego_controller_obj.go#L317

Reply:
@holisticode OK, if the operator ignores these ports and uses hard-coded ones then let's:
1) Export the hard-coded values (9650 and 9651) from the operator
2) Import them in the K8s runner